.github/workflows/windows-msvc-bdb.yml

```diff
@@ -147,8 +147,11 @@ jobs:
         run: |
           set COB_CFLAGS=/I "%cd%" /I "%VCPKG_ROOT%\installed\x64-windows\include"
           set COB_LIBS=libcob-5.lib /LIBPATH:"%cd%\build_windows\x64\Release"
+          set PATH=.\bin;%PATH%
           call "%VCVARS%"
-          cd tests
+          cd bin
+          cl gcdiff.c /I ..\build_windows /I ..\ ..\build_windows\x64\Release\libcob-5.lib
```

Comment:
Nice cheating for now (objously needs a separate commit directly to the repo); I'd still suggest to try using `cobc`, if possible (which possibly also gets OrangeC builds working with that definition).

Reply:
You didn't even give me the time to comment about this :D

Indeed I shall try using `cobc` instead of calling cl directly.

This (admittedly) hackish fix allows to pass more testsuite cases (now "only" ~75 fail).